### ISSUE TITLE
Fail to allocate bitmap

### ISSUE THREAD

**Issue by xinba6686** [open-mmlab/mmdetection#7681]
Thanks for your error report and we appreciate it a lot.

**Checklist**

1. I have searched related issues but cannot get the expected help.
2. I have read the [FAQ documentation](https://mmdetection.readthedocs.io/en/latest/faq.html) but cannot get the expected help.
3. The bug has not been fixed in the latest version.

**Describe the bug**

Fail to allocate bitmap

**Reproduction**

1. What command or script did you run?
python demo/video_demo.py configs/faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py checkpoints/faster_rcnn_r50_fpn_1x_coco_20200130-047c8118.pth --file demo/ligang/1.mp4  --out demo/ligang/result.mp4

```none
A placeholder for the command.
```

2. Did you make any modifications on the code or config? Did you understand what you have modified?  
NO.                                                                                
3. What dataset did you use?
COCO2014
**Environment**
addict	2.4.0	
albumentations	1.1.0	
appdirs	1.4.4	
asynctest	0.13.0	
atomicwrites	1.4.0	
attrs	21.4.0	
blas	2.113	
blas-devel	3.9.0	
ca-certificates	2021.10.8	
certifi	2021.10.8	
charset-normalizer	2.0.12	
cityscapesscripts	2.2.0	
click	8.1.1	
codecov	2.1.12	
colorama	0.4.4	
coloredlogs	15.0.1	
coverage	6.3.2	
cudatoolkit	11.1.1	
cycler	0.11.0	
cython	0.29.28	
flake8	4.0.1	
flatbuffers	2.0	
fonttools	4.31.2	
freetype	2.10.4	
humanfriendly	10.0	
idna	3.3	
imagecorruptions	1.1.2	
imageio	2.16.1	
importlib-resources	5.6.0	
iniconfig	1.1.1	
intel-openmp	2022.0.0	
interrogate	1.5.0	
isort	4.3.21	
jaraco-classes	3.2.1	
jaraco-collections	3.5.1	
jaraco-context	4.1.1	
jaraco-functools	3.5.0	
jaraco-structures	2.1.0	
jaraco-text	3.7.0	
jaraco-ui	2.3.0	
jaraco-windows	5.7.0	
joblib	1.1.0	
jpeg	9b	
kiwisolver	1.4.2	
kwarray	0.6.0	
libblas	3.9.0	
libcblas	3.9.0	
liblapack	3.9.0	
liblapacke	3.9.0	
libpng	1.6.37	
libtiff	4.2.0	
libuv	1.43.0	
libwebp	1.2.2	
libwebp-base	1.2.2	
libzlib	1.2.11	
lmdb	1.3.0	
lz4-c	1.9.3	
m2w64-gcc-libgfortran	5.3.0	
m2w64-gcc-libs	5.3.0	
m2w64-gcc-libs-core	5.3.0	
m2w64-gmp	6.1.0	
m2w64-libwinpthread-git	5.0.0.4634.697f757	
matplotlib	3.5.1	
mccabe	0.6.1	
mkl	2022.0.0	
mkl-devel	2022.0.0	
mkl-include	2022.0.0	
mmcv-full	1.4.8	
mmdet	2.23.0	
mmlvis	10.5.3	
mmpycocotools	12.0.3	
more-itertools	8.12.0	
msys2-conda-epoch	20160418	
networkx	2.7.1	
ninja	1.10.2.3	
numpy	1.22.3	
onnx	1.7.0	
onnxoptimizer	0.2.6	
onnxruntime	1.11.0	
opencv-contrib-python	4.5.5.64	
opencv-python	4.5.5.64	
opencv-python-headless	4.5.5.64	
openssl	1.1.1n	
ordered-set	4.1.0	
packaging	21.3	
path	16.4.0	
pillow	9.0.1	
pip	21.2.2	
pluggy	1.0.0	
protobuf	3.19.4	
py	1.11.0	
pycocotools	2.0.4	
pycodestyle	2.8.0	
pyflakes	2.4.0	
pyparsing	3.0.7	
pyquaternion	0.9.9	
pyreadline3	3.4.1	
pytest	7.1.1	
pytest-runner	6.0.0	
python	3.8.13	
python-dateutil	2.8.2	
python_abi	3.8	
pytorch	1.8.0	
pyturbojpeg	1.6.6	
pywavelets	1.3.0	
pyyaml	6.0	
qudida	0.0.4	
regex	2022.3.15	
requests	2.27.1	
scikit-image	0.19.2	
scikit-learn	1.0.2	
scipy	1.8.0	
setuptools	58.0.4	
six	1.16.0	
sqlite	3.38.0	
tabulate	0.8.9	
tbb	2021.5.0	
terminaltables	3.1.10	
threadpoolctl	3.1.0	
tifffile	2022.3.25	
tk	8.6.12	
toml	0.10.2	
tomli	2.0.1	
torchvision	0.9.0	
tqdm	4.63.1	
typing	3.7.4.3	
typing_extensions	4.1.1	
ubelt	1.0.1	
urllib3	1.26.9	
vc	14.2	
vs2015_runtime	14.27.29016	
wheel	0.37.1	
wincertstore	0.2	
xdoctest	1.0.0	
xz	5.2.5	
yapf	0.32.0	
zipp	3.7.0	
zlib	1.2.11	
1. Please run `python mmdet/utils/collect_env.py` to collect necessary environment information and paste it here.
sys.platform: win32
Python: 3.8.13 (default, Mar 28 2022, 06:59:08) [MSC v.1916 64 bit (AMD64)]
CUDA available: True
GPU 0: NVIDIA GeForce RTX 3090
CUDA_HOME: C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.1
NVCC: Not Available
GCC: n/a
PyTorch: 1.8.0
PyTorch compiling details: PyTorch built with:
  - C++ Version: 199711
  - MSVC 192829337
  - Intel(R) Math Kernel Library Version 2020.0.0 Product Build 20191125 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v1.7.0 (Git Hash 7aed236906b1f7a05c0917e5257a1af05e9ff683)
  - OpenMP 2019
  - CPU capability usage: AVX2
  - CUDA Runtime 11.1
  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_61,code=sm_61;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_7
5,code=sm_75;-gencode;arch=compute_80,code=sm_80;-gencode;arch=compute_86,code=sm_86;-gencode;arch=compute_37,code=compute_37
  - CuDNN 8.0.5
  - Magma 2.5.4
  - Build settings: BLAS_INFO=mkl, BUILD_TYPE=Release, CUDA_VERSION=11.1, CUDNN_VERSION=8.0.5, CXX_COMPILER=C:/cb/pytorch_1000000000000/work/tmp_bin/sccache-cl.exe, CXX_FLAGS=/DWIN32 /D_WINDOWS /GR /EHsc /w /bigobj -DUSE_PTHREADPOOL
 -openmp:experimental -DNDEBUG -DUSE_FBGEMM -DUSE_XNNPACK, LAPACK_INFO=mkl, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, TORCH_VERSION=1.8.0, USE_CUDA=ON, USE_CUDNN=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, US
E_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=OFF, USE_NNPACK=OFF, USE_OPENMP=ON,

TorchVision: 0.9.0
OpenCV: 4.5.5
MMCV: 1.4.8
MMCV Compiler: MSVC 192930141
MMCV CUDA Compiler: 11.1
MMDetection: 2.23.0+

2. You may add addition that may be helpful for locating the problem, such as
    - How you installed PyTorch [e.g., pip, conda, source]

I configure mmdetection on win10 and mmcv-full compiles on VS2019 for myself.

    - Other environment variables that may be related (such as `$PATH`, `$LD_LIBRARY_PATH`, `$PYTHONPATH`, etc.)
C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.29.30133\bin\Hostx86\x64
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.1\bin

**Error traceback**
If applicable, paste the error trackback here.

```none
A placeholder for trackback.
```

**Bug fix**
If you have already identified the reason, you can provide the information here. If you are willing to create a PR to fix it, please also leave a comment here and that would be much appreciated!
